src/Language/CSharp.ts

```diff
@@ -243,12 +243,15 @@ export class CSharpRenderer extends ConvenienceRenderer {
             return csType;
         }
     }
-
+    protected superclassForClass(): Sourcelike | undefined {
```

Comment:
This should take a `ClassType` as an argument (that's why it's called `xxxForClass`).  In fact, come to think of it, you might want superclasses for non-class types, too, like for unions.  So let's instead make it

```csharp
    protected superclassForType(_t: Type): Sourcelike | undefined
```

Reply:
I'll switch the method around, then.

Reply:
Switch it around?  I'm not sure what that means.

Reply:
Sorry. I meant change the arguments and name.